src/admin-guide/environments/gui.yml

```diff
@@ -0,0 +1,137 @@
+name: cylc-8-with-gui
+# Cylc 8 and the UI server installed from Conda
+# n.b. The UI needs to be installed with Yarn: For details see
+# https://github.com/cylc/cylc-ui
```

Comment:
The UI comes packaged with the UIS so we can skip this (also means we don't need NodeJS in Cylc environments).

```suggestion
```